berkeley-function-call-leaderboard/CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ All notable changes to the Berkeley Function Calling Leaderboard will be documen
   - `microsoft/Phi-3-mini-128k-instruct`
   - `microsoft/Phi-3-mini-4k-instruct`
 - [Sept 25, 2024] [#660](https://github.com/ShishirPatil/gorilla/pull/660): Bug fix in `parse_nested_value` function to handle nested dictionary values properly. 
+- [Sept 24, 2024] [#648](https://github.com/ShishirPatil/gorilla/pull/648): Add the following new models to the leaderboard:
+  - `gemini-1.5-pro-002`
+  - `gemini-1.5-pro-002-FC`
+  - `gemini-1.5-pro-preview-0514`
```

Comment:
Google now launched GA versions we don't need to include model with "preview" tag in the model name. So I think it is enough to include the below models:
1. gemini-1.5-flash-001 (stable version launched in May ) and gemini-1.5-flash-002 (stable version last week)
2.  gemini-1.5-pro-001 (stable version model launched in May ) and gemini-1.5-pro-002 (stable version last week)
3.  gemini-1.0-pro-001 (stable version model launched in Feb ) and gemini-1.0-pro-002 (stable version in Apr) 

All of the above models support function calling. Could we please add the above models to leaderboard (if not already included):

Sources: https://cloud.google.com/vertex-ai/generative-ai/docs/learn/model-versions#stable-versions-available and https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/function-calling#supported_models

Reply:
Sure. We can use the GA versions instead. 
(I was previously under the impression that the `preview` ones are better than the GA ones.)

Reply:
This PR looks good to me, will you update this PR with the GA models?

Reply:
Yes. I have updated the PR with the GA models.